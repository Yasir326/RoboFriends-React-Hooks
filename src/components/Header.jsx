import React from 'react';

const Header = () => {
  return <h1 className='f1'>Robodex</h1>;
};

export default Header;
