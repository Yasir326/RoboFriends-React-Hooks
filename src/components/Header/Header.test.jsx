import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';
import '../../setUpTest';

it('expect to render Card component', () => {
  expect(shallow(<Header />)).toMatchSnapshot();
});