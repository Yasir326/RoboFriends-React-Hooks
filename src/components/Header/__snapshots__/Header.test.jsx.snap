// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `
<h1
  className="f1"
>
  Robodex
</h1>
`;
